### PR TITLE
services/regulated-assets-approval-server: make sure the SEP-8 "pending" response contains the "timeout" field

### PR DESCRIPTION
**Pull request by marcelosalloum**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [x] This PR adds tests for the most critical parts of the new functionality or fixes.
* [x] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [x] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [x] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Make sure the SEP-8 "pending" response contains the "timeout" field.

### Why

According to [SEP-8](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0008.md#pending), the "timeout" field is mandatory and if no timestamp can be determined it should be set to "0":
> timeout is the (...) number of milliseconds to wait before submitting the same transaction again. Use `0` if the wait time cannot be determined.

### Known limitations

N/A

### CODE REVIEW

**Review by acharb — COMMENTED** [review]
code changes lgtm.

According to the [sep-8 docs](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0008.md#pending) it looks like the correct word is `timeout` instead of `timestamp` though 😄

**Comment by marcelosalloum** [comment]
Wow, my bad man 🤦‍♂️! Thanks for spotting that 🙏

File: services/regulated-assets-approval-server/internal/serve/tx_approve_response.go [hunk 1]
```diff
@@ -59,10 +60,12 @@ func NewSuccessTxApprovalResponse(tx, message string) *txApprovalResponse {
 }
 
 func NewPendingTxApprovalResponse(message string) *txApprovalResponse {
+	timestamp := int64(0)
```
Comment (acharb) [inline services/regulated-assets-approval-server/internal/serve/tx_approve_response.go:63]:
```suggestion
	timeout := int64(0)
```

**Review by acharb — APPROVED** [review]
noticed one var still needing change 😄  but lgtm!

File: services/regulated-assets-approval-server/internal/serve/tx_approve_response.go (hunk 1 above)
Comment (marcelosalloum) [reply services/regulated-assets-approval-server/internal/serve/tx_approve_response.go]: Thanks again! Fixed in https://github.com/stellar/go/pull/3681/commits/ef284d779329a24ed3dc71d8293db3661675fd2e

**Pull request merged**